test_main.py

```diff
@@ -19,13 +22,24 @@ def message(self):
 
     return MockResponse()
 
+@pytest.fixture(autouse=True)
+def set_openai_api_key():
+    with patch.dict(os.environ, {"OPENAI_API_KEY": "test"}):
+        yield
+
+
 def test_generate_headline():
     text = "This is a sample website text."
     image_url = "http://example.com/image.jpg"
-    with patch("openai.ChatCompletion.create", side_effect=mock_openai_chat_completion_create):
+    with patch("app.openai_utils.get_openai_client") as mock_get_client:
```

Comment:
Can you make this openai mocking into its own method?

Reply:
Certainly! We can refactor the OpenAI mocking into its own method to keep the test code clean and reusable. I'll make the necessary changes and update the PR.